examples/main/layer-examples.js

```diff
@@ -381,7 +369,6 @@ export default {
   'Core Layers': {
     'GeoJsonLayer': GeoJsonLayerExample,
     'GeoJsonLayer (Extruded)': GeoJsonLayerExtrudedExample,
```

Comment:
They are using different datasets. If we can find a good data that contains all feature types, I don't see why not.